src/scss/components/file/_index.scss

```diff
@@ -0,0 +1,65 @@
+@import "components/form/config";
+@import "components/box/index";
+@import "hx-file-control";
+
+hx-drop-fence {
```

Comment:
Given that `hx-file-control` has its own partial, let's do the same for all of the other elements below.

Split into `_hx-drop-fence.scss` partial.